webapp/packages/plugin-sql-editor/src/SqlEditorSettingsService.ts

```diff
@@ -6,21 +6,34 @@
  * you may not use this file except in compliance with the License.
  */
 import { Dependency, injectable } from '@cloudbeaver/core-di';
-import { createSettingsAliasResolver, PluginManagerService, PluginSettings, SettingsManagerService } from '@cloudbeaver/core-plugin';
+import {
+  createSettingsAliasResolver,
+  ESettingsValueType,
+  PluginManagerService,
+  PluginSettings,
+  SettingsManagerService,
+} from '@cloudbeaver/core-plugin';
 import { ServerSettingsResolverService, ServerSettingsService } from '@cloudbeaver/core-root';
 import { schema } from '@cloudbeaver/core-utils';
 
+import { SQL_EDITOR_SETTINGS_GROUP } from './SQL_EDITOR_SETTINGS_GROUP';
+
 const defaultSettings = schema.object({
   maxFileSize: schema.coerce.number().default(10 * 1024), // kilobyte
   disabled: schema.coerce.boolean().default(false),
   autoSave: schema.coerce.boolean().default(true),
 });
 
+const defaultProposalInsertTableSettings = schema.object({
+  alias: schema.coerce.boolean().default(true),
```

Comment:
please use boolean name, `aliasesEnabled` or something like this

Reply:
`sql.proposals.insert.table.alias` - this is a pretty strict setting name I cannot rename cause it set on the backend side of the app. and it should be consistent to dbeaver desktop analogue setting

in the code of course it is used with the boolean prefix:
https://github.com/dbeaver/cloudbeaver/pull/2370/files#diff-603b6bb92ed26d86a6494be8e6ba52611dd550d3754f8bb9ed8b22ce7fa9ec81R42